orte/mca/errmgr/errmgr.h

```diff
@@ -119,6 +119,10 @@ typedef int (*orte_errmgr_base_module_abort_peers_fn_t)(orte_process_name_t *pro
                                                         orte_std_cntr_t num_procs,
                                                         int error_code);
 
+typedef void (*orte_errmgr_base_module_start_error_detector_fn_t)(void);
```

Comment:
What is this ?

Reply:
This should be deleted, this is an old version of enable_detector.  deleted.